src/components/UserMenu/UserMenuSignedIn.tsx

```diff
@@ -0,0 +1,214 @@
+                '_blank',
+                'noopener'
+              )
+        },
+        onClickMenuItem ? 300 : 0
+      )
+    },
+    [onClickMyAssets, onClickMenuItem, trackingId]
+  )
+
+  const handleClickAccountSettings = useCallback(
+    (event: React.MouseEvent<HTMLElement, MouseEvent>) => {
+      onClickMenuItem &&
+        onClickMenuItem(event, UserMenuLabels.SETTINGS, trackingId)
+
+      setTimeout(
+        () => {
+          onClickAccountSettings
+            ? onClickAccountSettings(event)
+            : window.open(
+                `${config.get('MARKETPLACE_SETTINGS_URL')}`,
+                '_blank',
+                'noopener'
+              )
+        },
+        onClickMenuItem ? 300 : 0
+      )
+    },
+    [onClickAccountSettings, onClickMenuItem, trackingId]
+  )
+
+  const handleClickProfile = useCallback(
+    (event: React.MouseEvent<HTMLElement, MouseEvent>) => {
+      onClickMenuItem &&
+        onClickMenuItem(event, UserMenuLabels.PROFILE, trackingId)
+
+      setTimeout(
+        () => {
+          onClickProfile
+            ? onClickProfile(event)
+            : window.open(config.get('PROFILE_URL'), '_blank', 'noopener')
+        },
+        onClickMenuItem ? 300 : 0
+      )
+    },
+    [onClickProfile, onClickMenuItem, trackingId]
+  )
+
+  const handleClickWallet = useCallback(
+    (event: React.MouseEvent<HTMLElement, MouseEvent>) => {
+      onClickMenuItem &&
+        onClickMenuItem(event, UserMenuLabels.WALLET, trackingId)
+
+      setTimeout(
+        () => {
+          onClickWallet
+            ? onClickWallet(event)
+            : window.open(config.get('ACCOUNT_URL'), '_blank', 'noopener')
+        },
+        onClickMenuItem ? 300 : 0
+      )
+    },
+    [onClickWallet, onClickMenuItem, trackingId]
+  )
+
+  const handleClickSignOut = useCallback(
+    (event: React.MouseEvent<HTMLElement, MouseEvent>) => {
+      onClickSignOut(event, trackingId)
+    },
+    [onClickSignOut, trackingId]
+  )
+
+  const handleClickToggle = useCallback(
+    (event: React.MouseEvent<HTMLElement, MouseEvent>) => {
+      onClickToggle(event)
+    },
+    [onClickToggle, trackingId]
+  )
+
+  return (
+    <div className={classNames('dcl', 'user-menu-sign-in')}>
+      {notifications && <Notifications {...notifications} />}
+      <Button basic onClick={handleClickActivity} className="activity-icon">
+        <ActivityIcon hasActivity={hasActivity} />
+      </Button>
+      <ManaBalances
+        manaBalances={manaBalances}
+        onClickBalance={onClickBalance}
+      />
+      <div className="toggle" onClick={handleClickToggle}>
+        <AvatarFace size="medium" avatar={avatar} />
+      </div>
+      <div
+        className={classNames(
+          'menu-wrapper',
+          isOpen && 'open',
+          isClickable && 'clickable'
+        )}
+      >
+        <div className="menu-wearable-preview">
+          <img src={avatar.avatar.snapshots.body} />
```

Comment:
Users can be logged in and not have a profile. What will be shown instead?

Reply:
done